PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/ruleConfigs.json

```diff
@@ -1,6 +1,12 @@
 {
-  "StorageUploadWithoutLengthCheck": {
-    "methodsToCheck": [
+  "HardcodedAPIKeysAndTokensCheck": {
+    "clients_to_check": ["AzureKeyCredential", "AccessToken"],
+    "anti_pattern_message": "Use of API keys or tokens in code is not recommended. We recommend Secretless authentication through DefaultAzureCredential.",
```

Comment:
Better to phrase this along these lines 
DefaultAzureCredential is recommended for authentication if the service client supports Token Credential (Entra ID Authentication). if not, then use Azure Key Credential for API key based authentication.

Reply:
done:)